mw-webapp/src/styles/_variables.scss

```diff
@@ -12,4 +12,9 @@ $link-active: rgb(255, 149, 0);
 $header-background: rgb(85, 122, 149);
 
 // Title
-$title-padding: 10px;
\ No newline at end of file
+$title-padding: 10px;
+
+// Input
+$input-border-primary: rgb(77, 77, 77);
+$input-border-secondary: #373434;
+$input-focus: rgb(226, 166, 109);
```

Comment:
Use always or rgb() or # colors. When task about design will be ready we will have conventions about styles but now  need to stick to the same type